client/state/selectors/test/get-plugin-upload-error.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import { getPluginUploadError } from 'state/selectors';
+
+const siteId = 77203074;
+const error = {
+	error: 'folder_exists',
+	message: 'folder_exists',
+};
+
+describe( 'getPluginUploadError', () => {
+	it( 'should return null by default', () => {
+		const state = deepFreeze( {
```

Comment:
What's the benefit of `deepFreeze`-ing state for testing selectors? Makes sense to verify prevention of state mutation when testing reducers, but for selectors?